doc/releases/changelog-dev.md

```diff
@@ -30,6 +30,12 @@
 
 <h3>Deprecations 👋</h3>
 
+* The boolean functions provided by ``pennylane.operation`` are deprecated. See each function's docstring
+  for equivalent code. These include ``not_tape``, ``has_gen``, ``has_grad_method``, ``has_multipar``,
+  ``has_nopar``, ``has_unitary_gen``, ``is_measurement``, ``is_trainable``,
+  ``defines_diagonalizing_gates``, and ``gen_is_multi_term_hamiltonian``.
```

Comment:
```suggestion
* The following boolean functions in ``pennylane.operation`` are now deprecated: :func:`~.operation.not_tape`, 
  :func:`~.operation.has_gen`, :func:`~.operation.has_grad_method`, :func:`~.operation.has_multipar`, :func:`~.operation.has_nopar`, :func:`~.operation.has_unitary_gen`, 
  :func:`~.operation.is_measurement`, :func:`~.operation.`is_trainable`, ``defines_diagonalizing_gates``, and ``gen_is_multi_term_hamiltonian``. 
   Please consult each function's docstring for alternative code. 
   [(#7319)](https://github.com/PennyLaneAI/pennylane/pull/7319)
```

Reply:
Didn't know we could use sphinx directives in the markdown file 🚀 . Good thing to learn.  Might even be worth circulating this piece of information more widely.

Reply:
Would this also be the fix to current sphinx [complaints](https://productionresultssa6.blob.core.windows.net/actions-results/e18fcf5e-289e-4781-a23f-53dc35f9bb6a/workflow-job-run-5763488f-005a-5e4b-e433-74c6d989d315/logs/job/job-logs.txt?rsct=text%2Fplain&se=2025-04-28T19%3A25%3A51Z&sig=V4G6W%2Bvm3tOdgETE6MICg%2Bqjtuk4vG%2BzGKNvvV%2FBRKs%3D&ske=2025-04-29T06%3A02%3A26Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2025-04-28T18%3A02%3A26Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2025-01-05&sp=r&spr=https&sr=b&st=2025-04-28T19%3A15%3A46Z&sv=2025-01-05)?